specs/_features/eip7594/das-core.md

```diff
@@ -195,27 +195,34 @@ def recover_matrix(partial_matrix: Sequence[MatrixEntry],
 
 ```python
 def get_data_column_sidecars(signed_block: SignedBeaconBlock,
-                             blobs: Sequence[Blob]) -> Sequence[DataColumnSidecar]:
+                             cells_and_kzg_proofs: Sequence[Tuple[
```

Comment:
Hmm. Why do we pass `cells_and_kzg_proofs` all together and we don't pass `cells` and `proofs` as separate inputs? Feels more natural and closer to the rest of the codebase.

Reply:
Yeah that's actually what I started doing at first, but decided against it when I started using it. Passing it together allows us to store the results of `compute_cells_and_kzg_proofs` without splitting it into two lists. It also saves of from a few length checks, ensuring there are equal numbers of cells/proofs.

See this:

https://github.com/ethereum/consensus-specs/blob/8da90b27020813d0a90abc156d68446984fe37a1/tests/core/pyspec/eth2spec/test/eip7594/merkle_proof/test_single_merkle_proof.py#L41-L42

Reply:
> It also saves of from a few length checks, ensuring there are equal numbers of cells/proofs

That's the main reason in my mind.